lib/embulk/input/mixpanel.rb

```diff
@@ -71,7 +72,8 @@ def self.transaction(config, &control)
           incremental: incremental,
           slice_range: config.param(:slice_range, :integer, default: 7),
           job_start_time: job_start_time,
-          incremental_column_upper_limit: incremental_column_upper_limit
+          incremental_column_upper_limit: incremental_column_upper_limit,
+          allow_partial_import: config.param(:allow_partial_import,:bool, default: true)
```

Comment:
Is Yao aware of this default `true`? Apparently, I thought `false` would be expected.

Reply:
the current default behavior is don't do anything. We want to keep it the same.